resources/views/submission/submission.blade.php

```diff
@@ -47,6 +54,9 @@
                         <td>
                             <a href="{{route('countySubmission')."/".$county->id}}" class="btn btn-primary"> View </a>
                         </td>
+                        <td>
+                            <a href="{{route('submissionEdit')."/".'County'."/".$county->id}}" class="btn btn-primary"> Edit </a>
+                        </td>
                         <td>Pending</td>
                     </tr>
                 @endforeach
```

Comment:
For each of the $approved ForEach loops, could you also add a Column in the row - in the place where the edit button would usually be. Just have this Icon (https://fontawesome.com/icons/lock?style=solid) there instead of the edit button.